src/components/PageTitle.scss

```
@@ -0,0 +1,22 @@
+@import '../styles/colors.scss';
+@import '../styles/textStyles.scss';
+
+.page-title {
+    @include title01-heavy;
+    color: $default-color-system-blue;
+    display: flex;
+    width: calc(100vw - 40px);
+    padding: 0px 20px 0px 20px;
```

```suggestion
    padding: 0 20px 0 20px;
```

Unit of measure 'px' is redundant.
보통 값이 0이라면 단위는 생략합니다.